geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/index/HBaseFeatureIndex.scala

```diff
@@ -79,10 +80,7 @@ trait HBaseFeatureIndex extends HBaseFeatureIndexType with ClientSideFiltering[R
       if (!desc.getCoprocessors.contains(name)) {
         // TODO: Warn if the path given is different from paths registered in other coprocessors
         // if so, other tables would need updating
-        coprocessorUrl match {
-          case Some(path) => desc.addCoprocessor(name, path, Coprocessor.PRIORITY_USER, null)
```

Comment:
Do we still need this check? Not sure what the reflection is going to do if you don't have a url/path.

Reply:
I moved it to the API shim - it takes an `Option[Path]` and uses default values that correspond to the default values in `HTableDescriptor.addCoprocessor(String)` (which calls `HTableDescriptor.addCoprocessor(String, Path, Int, Map) with PRIORITY_USER and nulls)`.

Reply:
https://github.com/elahrvivaz/geomesa/blob/13f23f98524f0138b1c2da7e9919002d2ea334cb/geomesa-hbase/geomesa-hbase-datastore/src/main/scala/org/locationtech/geomesa/hbase/utils/HBaseVersions.scala#L25